.vscode/tasks.json

```diff
@@ -3,6 +3,11 @@
     // for the documentation about the tasks.json format
     "version": "2.0.0",
     "tasks": [
+        {
+            "label": "(Docker) Elixir 1.10 IEx",
+            "type": "shell",
+            "command": "./iex.sh",
+        },
```

Comment:
Misschien 1.9 nog ff weghalen hier?

Reply:
1.7 ook dan staat er nog verder onder?

Reply:
Ik denk dat het handig is om de twee "buitenste" versies die we ondersteunen te laten staan, de tussenliggende versies zullen waarschijnlijk sowieso wel werken denk ik